lib/block-supports/dimensions.php

```diff
@@ -10,21 +10,43 @@
  *
  * @param WP_Block_Type $block_type Block Type.
  */
-function gutenberg_register_spacing_support( $block_type ) {
-	$has_spacing_support = gutenberg_block_has_support( $block_type, array( 'spacing' ), false );
-
+function gutenberg_register_dimensions_support( $block_type ) {
```

Comment:
I have a second thought about this PR: I'm unsure whether we can rename a function that's been already shipped to WordPress core ([see](https://github.com/WordPress/wordpress-develop/blob/master/src/wp-includes/block-supports/spacing.php)). Same for renaming the file: `spacing.php` to `dimensions.php`. These become "public APIs" once shipped and, generally, we can't remove them without notice: there's a deprecation process if need be, etc.

Would like thoughts from folks with more experience working with WordPress core: @gziolo @jorgefilipecosta @youknowriad

Reply:
I agree, it's similar to the above comment, even if this functions is marked "internal" in Core, so in theory we could rename, I think the file should be kept as `spacing.php` and the function renamed... Just stay consistent as long as the API is "spacing".

Reply:
I'm fine using spacing for all.

Reply:
I'd be in the camp of using "Dimensions" for the UI labels... but keeping "spacing" for the code because of the backward compatibility concerns.